Sources/AnimalKingdomAPI/AnimalKingdomAPI/Sources/Fragments/ClassroomPetDetails.graphql.swift

```diff
@@ -212,7 +212,7 @@ public struct ClassroomPetDetails: AnimalKingdomAPI.SelectionSet, Fragment {
   /// Parent Type: `Bird`
   public struct AsBird: AnimalKingdomAPI.InlineFragment {
     public let __data: DataDict
-    public init(data: DataDict) { __data = data }
+    public init(_dataDict: DataDict) { __data = _dataDict }
```

Comment:
Was this underscored param in a public api supposed to be unnamed, and there’s a missing space? Otherwise seems a strange change. Is it to deter people from using it?

Reply:
Yes, this is to indicate that it's an internal API that needs to be `public` only because it's used by generated code. You should not be calling it yourself. 

In my next PR, I'll be adding a warning to the documentation of this initializer that it should not be called outside of generated code.